pkg/aggregator/aggregator.go

```diff
@@ -36,6 +37,22 @@ const bucketSize = 10                         // fixed for now
 // MetricSamplePoolBatchSize is the batch size of the metric sample pool.
 const MetricSamplePoolBatchSize = 32
 
+// hugeSeriesTagThreshold is the number of tags we consider "huge" and a sign
+// that something is probably wrong with the series.  If changing this value,
+// also change the description in tlmHugeSeries.
+const hugeSeriesTagThreshold = 100
```

Comment:
Do we want to make those configurable ?

Reply:
@olivielpeau -- what do you think?

Reply:
No I don't think we should make these configurable, this is just for troubleshooting.